app/api/review.py

```
@@ -0,0 +1,352 @@
+        rating=review_data.rating,
+        review_text=review_data.review_text,
+    )
+    db.add(review)
+    await db.commit()
+    return await _create_or_update_review_response(review, db)
+
+
+@router.get("/", response_model=ReviewResponse)
+async def list_reviews(
+    current_user: Annotated[User, Depends(get_current_user)],
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)